static/src/javascripts/projects/common/modules/commercial/dfp/dfp-obj.js

```diff
@@ -0,0 +1,4 @@
+define(function () {
```

Comment:
That's a fair point, populating the `dfp` map should be the responsibility of the `dfp-api` module, not each submodule. In reality, the `dfp` map is completely useless. I can make without it. Let's see.